hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -839,54 +954,103 @@
       tests: []
     }
 
-    // CSRNG tests:
+    // ENTROPY_SRC (pre-verified IP) integration tests:
     {
-      name: chip_csrng_cmd
-      desc: '''Verify the cmd interface to CSRNG.
+      name: chip_entropy_src_ast_rng_req
+      desc: '''Verify the RNG req to ast.
 
-            Details TBD. SW test validates the reception of cmd req done interrupt.
+            Details TBD.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_csrng_entropy_src
-      desc: '''Verify the interface to entropy_src.
+      name: chip_entropy_src_ast_fips
+      desc: '''Verify the connectivity of rng_fips_o feedback signal to RNG.
 
             Details TBD.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_csrng_fuse
-      desc: '''Verify the fuse input to CSRNG.
+      name: chip_entropy_src_csrng
+      desc: '''Verify the transfer of entropy bits to CSRNG.
 
             Details TBD.
             '''
       milestone: V2
       tests: []
     }
-
-    // ENTROPY_SRC (pre-verified IP) integration tests:
     {
-      name: chip_entropy_src_ast_rng_req
-      desc: '''Verify the RNG req to ast.
+      name: chip_entropy_src_cs_aes_halt
+      desc: '''Verify the aes halt handshake with CSRNG.
 
-            Details TBD. SW test validates the reception of the entropy valid interrupt.
+            Details TBD.
             '''
       milestone: V2
       tests: []
     }
     {
-      name: chip_entropy_src_fuse
+      name: chip_entropy_src_fuse_en_fw_read
       desc: '''Verify the fuse input entropy_src.
 
             Details TBD.
             '''
       milestone: V2
       tests: []
     }
+    {
+      name: chip_entropy_src_xht
+      desc: '''Verify the entropy_src extended health test interface.
+
+            Details TBD.
+            '''
+      milestone: V2
+      tests: []
+    }
+
+    // CSRNG tests:
+    {
+      name: chip_csrng_edn_cmd
+      desc: '''Verify incoming command interface from EDN.
+
+            Verify that there are no misconnects between each EDN requesting RNG bits from CSRNG.
+            Details TBD.
```

Comment:
Each EDN block can be setup to request from CSRNG. The command done interrupt will indicate that the command was completed, thus the request bus connected properly. There is no way to know if the data return bus is working, maybe need to add some status bit.